src/azure-cli/azure/cli/command_modules/managedservices/_help.py

```diff
@@ -0,0 +1,174 @@
+"""
+
+helps['managedservices definition assignment show'] = """
+    type: command
+    short-summary: Gets a registration assignment.
+    long-summary: Gets the registration assignment given its identifier (guid) or the fully qualified resource id. When resource id is used, subscription id and resource group parameters are ignored.
+    parameters:
+        - name: --assignment
+          short-summary: |
+             The identifier (guid) or the fully qualified resource id of the registration assignment. When resource id is used, subscription id and resource group parameters are ignored.
+    examples:
+        - name: Get an assignment given its identifier under the default subscription scope.
+          text: az managedservices definition assignment show --assignment d3087cf0-e180-4cca-b147-54ae00c7b504
+        - name: Get an assignment given its fully qualified resource id.
+          text: az managedservices definition assignment show --assignment /subscriptions/a62076fa-768a-403c-9d9d-6a9919aae441/providers/Microsoft.ManagedServices/registrationAssignments/0c3e9687-b461-4615-b6e4-74d54998d6e4
+        - name: Get an assignment given its identifier under the default subscription scope with the registration definition details.
+          text: az managedservices definition assignment show --assignment d3087cf0-e180-4cca-b147-54ae00c7b504 --include-definition true
+"""
+
+helps['managedservices definition assignment delete'] = """
+    type: command
+    short-summary: Deletes the registration assignment.
+    long-summary: Deletes the registration assignment given its identifier (guid) or the fully qualified resource id.
+    parameters:
+        - name: --assignment
+          short-summary: |
+             The identifier (guid) or the fully qualified resource id of the registration assignment. When resource id is used, subscription id and resource group parameters are ignored.
+    examples:
+        - name: Deletes an assignment given its identifier under the default subscription scope.
+          text: az managedservices definition assignment delete --assignment d3087cf0-e180-4cca-b147-54ae00c7b504
+        - name: Deletes an assignment given its fully qualified resource id.
+          text: az managedservices definition assignment delete --assignment /subscriptions/a62076fa-768a-403c-9d9d-6a9919aae441/providers/Microsoft.ManagedServices/registrationAssignments/0c3e9687-b461-4615-b6e4-74d54998d6e4
+"""
+
+helps['managedservices definition assignment list'] = """
+    type: command
+    short-summary: List all the registration assignments.
+    long-summary: List all the registration assignments. Subscription id and resource group parameters can be used to override default values.
```

Comment:
The long summary should not duplicate the short summary. The CLI essentially concatenates the two so this will result in duplicate lines.

Reply:
Understood. I changed all the long summaries to short and remove the long ones.